helm/reana/templates/reana-workflow-controller.yaml

```diff
@@ -91,6 +91,8 @@ spec:
           - name: REANA_DEFAULT_QUOTA_DISK_LIMIT
             value: {{ .Values.quota.default_disk_limit | default 0 | quote }}
           {{- if .Values.quota.enabled }}
+          - name: REANA_INGRESS_HOST
```

Comment:
The `REANA_INGRESS_HOST` should not be located inside the "if quota..." clause, because it does not depend on quotas. Please move it above, next to where `REANA_INGRESS_ANNOTATIONS` lives.